tests/test_validation_framwork/test_data_manager.py

```diff
@@ -206,7 +207,7 @@ def test_DataManager_dataset_names():
 def test_DataManager_get_data():
 
     datasets = setup_TestDatasets()
-    dm = DataManager(datasets, 'DS1')
+    dm = DataManager(datasets, 'DS1', read_ts_names={f'DS{i}': 'read' for i in range(1,4)})
```

Comment:
- Most warnings we had were actually not due to code in pytesmo but in the reader packages (ascat, ismn) in the tests
  - pytesmo by default uses the ``read_ts`` function of the readers, which is deprecated and therefore prints a warning.
  - All readers would already work using ``read()`` instread of ``read_ts()`` except for ISMN, I opened an issue therefore  https://github.com/TUW-GEO/ismn/issues/26